configuration/wireless/access-point.md

```diff
@@ -70,7 +70,7 @@ You need to edit the hostapd configuration file, located at /etc/hostapd/hostapd
 sudo nano /etc/hostapd/hostapd.conf
 ```
 
-Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of NameOfNetwork, and a password AardvarkBadgerHedgehog. Note that the name and password should **not** have quotes around them. 
+Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of NameOfNetwork, and a password AardvarkBadgerHedgehog. Note that the name and password should **not** have quotes around them. The passphrase should be between 8 and 64 charaters in length.
```

Comment:
Typo
charaters

Reply:
@SurferTim Did you see these comments?

Reply:
That is correct. In hostapd.conf, the ssid and passphrase should not have quotes around them. Actually, the passphrase should be no less than 8 characters and no more than 63.

Reply:
I was thinking more of the spelling mistakes...